Server/src/models/interfaces/DataRequestModelInterface.ts

```diff
@@ -6,4 +6,5 @@ export interface IDataRequestModel {
     year: number
     categoryId: number
     subcategoryId: number
+    datapoint: string[]

```

Comment:
Whats being tracked here? Datapoints are key value pairs with datapoint name as key and datapoint numbers as value. Whats supposed to be in this datapoint?

If this is searching, shouldn't it be 1 string? Since FE will only allowing searching by 1 datapoint name at a time

Reply:
Is it only one datapoint name at a time? The story makes reference to searching by multiple names at a time.

Reply:
Its a dropdown of names so it should only be 1 at a time

Reply:
in that case, should I revert the array of datapoints to single datapoint inside the datasetService file?

Reply:
deleted the array symbols and modified the service file to have single datapoint on the  recent commit